### PR TITLE
Created Species List Component

### PR DESCRIPTION
**Pull request by wangscs**
Initial creation with first test "should create"

### CODE REVIEW

**Comment by arthuranteater** [comment]
test ok comments?

**Comment by mosdevly** [comment]
You have a failing test on this build.

Also, you can ping your teammates if you need a code review from them because maybe they haven't seen it yet.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterModule, Routes } from '@angular/router';
+import { SpeciesListComponent } from './components/species-list/species-list.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: 'species',
```
Comment (mosdevly) [inline src/app/app-routing.module.ts:8]: We have a route with no test.

**Review by mosdevly — CHANGES_REQUESTED** [review]
Needs a route test.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (wangscs) [reply src/app/app-routing.module.ts:8]: Route test for species has been added to nav test component

**Review by mosdevly — CHANGES_REQUESTED** [review]
Here's an example of how to test your router: https://github.com/gSchool/ng-rickdiculous-app/blob/solution/src/app/components/side-menu/side-menu.component.spec.ts

Keep in mind that there is an existing test file where route tests should go: https://github.com/Tech-Student-Labs/swapi-client-ts/blob/master/src/tests/routes.spec.ts

**Review by wangscs — COMMENTED** [review]
Added another route test for species under routes.spec

**Review by mosdevly — APPROVED** [review]

**Pull request merged**